dist/index.html

```diff
@@ -8,10 +8,30 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
     
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
```

Comment:
whoa lots of merge conflict tags here

when  there's a conflict in `dist/` files in general  I usually just regenerate them with `yarn run build-production` since they're auto generated

Reply:
oh ok! I thought it would be auto regenerated after I resolve! I will do ```run build-production``` :)